backend/benefit/applications/services/pdf_templates/composed_accepted_public.html

```diff
@@ -1,10 +1,10 @@
 {% extends "base.html" %}
 
 {% block content %}
-<p>Työllisyydenhoidon Helsinki-lisän myöntäminen työnantajille</p>
+<p>Työllisyydenhoidon Helsinki-lisä myöntäminen työnantajille</p>
```

Comment:
Same as https://github.com/City-of-Helsinki/yjdh/pull/1215/files#r943182821 i.e. former version is grammatically correct and the new one is not.

Reply:
no idea why, now fixed